integrations/tensorflow/bindings/python/pyiree/tf/support/tf_utils.py

```diff
@@ -28,8 +29,6 @@
 from pyiree.tf import compiler
 import tensorflow.compat.v2 as tf
 
-flags.DEFINE_bool("keep_saved_model", False,
```

Comment:
is there any reason to remove this? It seems useful to be able to get it out.

Reply:
There is, but it's mostly because the way we handle parallel tests and writing `SavedModel`s to disk is hacky. We currently need to create a `SavedModel` for each backend if `--keep_saved_model` is passed to `bazel test`.

I could keep it as it is for now, and then simplify it when we move to using Bazel's undeclared test outputs?

Reply:
Oh okay. Seems fine either way.

Reply:
Sounds good. I'm going to get rid of it for the time being and add it back in as the default behavior for people who want to have a copy of the test artifacts.